core/src/wsv.rs

```diff
@@ -636,11 +770,10 @@ impl WorldStateView {
     }
 
     /// Initializes WSV with the blocks from block storage.
-    #[iroha_futures::telemetry_future]
-    pub async fn init(&self, blocks: Vec<VersionedCommittedBlock>) {
+    pub fn init(&mut self, blocks: Vec<VersionedCommittedBlock>) {
         for block in blocks {
             #[allow(clippy::panic)]
-            if let Err(error) = self.apply(block).await {
+            if let Err(error) = self.apply(block) {
                 error!(%error, "Initialization of WSV failed");
```

Comment:
Do we really need both? What happens if we just `.expect`.

Reply:
You are right this should be expect.